css/css-fonts/variations/font-variation-settings-parsing.html

```diff
@@ -35,6 +35,7 @@
         valueParserTests.forEach(function (testCase) {
             test(() => {
                 var element = document.getElementById("value-parser-test");
+                element.style.fontVariationSettings = "";
```

Comment:
Why is this needed?

Reply:
The last two failures in https://crbug.com/809950 were not actual failures -- they failed because the last test failed (leading to no resetting of font-variation-settings). Since they were invalid parsing tests, they relied on font-variation-settings not being set prior to the test. While the first commit fixes the test for Chrome, this would fix it for other browsers which might run into the same problem where one test fails immediately before an invalid parsing test.

Reply:
Thanks for the clarification and finding that out. Perhaps it's useful adding a comment here, like: "Reset to empty in order to allow testing to continue in case of parse errors occuring for the next test value."

Reply:
Good suggestion, I added that.